source/messaging/get-started-mattermost-channels.rst

```diff
@@ -0,0 +1,102 @@
+Get Started with Mattermost Channels
+====================================
+
+Access Mattermost Channels by selecting **Channels** from the **Product list** |product-list| icon.
+
+.. |product-list| image:: ../images/products_E82F.svg
+  :height: 24px
+  :width: 24px
+  :alt: Navigate between Channels, Playbooks, and Boards using the Product list icon.
+
+Channels are where you can communicate with your team about various topics or projects. 
+Channels and channel management tools are available in the channel sidebar to the left. 
+
+See [XXX] for details on the many ways you can use Channels to enable team collaboration.
+
+Channels and direct messages
+----------------------------
+
+Everyone who joins a Mattermost workspace is automatically added to two default channels: **Town Square** and **Off-Topic**. 
+
+- **Town Square** is great for your team-wide communications, such as announcements.
+- **Off-Topic** is great for fun, social, non-work-related conversations. 
+
+Direct messages are private conversations between two people. 
+Group messages are private conversations for small groups of up to seven people. 
+Start a new direct or group message by selecting the **Plus** |plus| icon next to **Direct Messages** in the channel sidebar.
+
+.. |plus| image:: ../images/plus_F0415.svg
+  :height: 24px
+  :width: 24px
+  :alt: The Plus icon provides access to channel and direct message functionality.
+
+.. tip:: 
+    Direct and group messages are also channels in Mattermost, so many of the [preferences available for channels] are also available for direct messages. 
+
+Create and join channels
+------------------------
+
+Channels are either **public** or **private**.
+
+- **Public** channels are identified with a **Globe** |globe| icon. Anyone on the team can join a public channel.
+- **Private** channels are typically used for sensitive topics, and are identified with a **Lock** |lock| icon. You must be invited to private channels by another channel member.  
+
+.. |globe| image:: ../images/globe_E805.svg
+  :height: 24px
+  :width: 24px
+  :alt: Public channels are identified with a Globe icon.
+
+.. |lock| image:: ../images/lock-outline_F0341.svg
+  :height: 24px
+  :width: 24px
+  :alt: Private channels are identified with a Lock icon.
+
+To see all available public channels you can join, select the **Plus** |plus| icon at the top of the channel sidebar, then select **Browse Channels**. Search for channels by name or scroll through the list. Select **Join** next to any channel to become a member of that channel.
+
+.. tip:: 
+    Select **Find Channel** in the channel sidebar to see all of the channels you're currently a member of across all of your teams, including public and private channels, as well as all direct and group messages.
```

Comment:
The find channels doesn't actually display all your channels. It lists your recently visited channels and helps you jump to channels quickly by typing their name